### PR TITLE
Prepare v022

### PR DESCRIPTION
**Pull request by jvanhoefer**
Fixes some broken links/typos and prepares the release of v0.2.2

### CODE REVIEW

File: doc/release_notes.rst [hunk 1]
```diff
@@ -5,6 +5,13 @@ Release Notes
 0.2 series
 ..........
 
+0.2.2 (2021-03-19)
+------------------
+
+* Add checks in SBML conversion (e.g. to catch invalid identifiers and equations).
```
Comment (yannikschaelte) [inline doc/release_notes.rst:11]: optionally add (# commit id)

File: doc/release_notes.rst [hunk 2]
```diff
@@ -5,6 +5,13 @@ Release Notes
...
+* Add tests for MacOs and Windows.
```
Comment (yannikschaelte) [inline doc/release_notes.rst:12]:
```suggestion
* Add tests for MacOS and Windows.
```

**Review by yannikschaelte — APPROVED** [review]

File: doc/release_notes.rst (hunk 1 above)
Comment (jvanhoefer) [reply doc/release_notes.rst:11]: Thought about it, but was not done so far, so I preferred to be consistent :)

Comment (jvanhoefer) [reply doc/release_notes.rst:11]: Might be something to add also for v0.2.1 at some point for the next release :)

Comment (jvanhoefer) [reply doc/release_notes.rst]: Ok, changed my mind and added them :D

**Pull request merged**